JSDemos/Demos/TreeView/DragAndDropPlainDataStructure/React/App.tsx

```diff
@@ -18,7 +18,7 @@ const calculateToIndex = (e) => {
     : e.toIndex + 1;
 };
 
-const findNode = (treeView, index) => {
```

Comment:
In places like this its strange for me, that i cant easily get instance type from component